src/coreclr/vm/gcinfodecoder.cpp

```diff
@@ -2093,7 +2095,7 @@ void GcInfoDecoder::ReportRegisterToGC(
     LOG((LF_GCROOTS, LL_INFO1000, "Reporting " FMT_REG, regNum ));
 
     OBJECTREF* pObjRef = GetRegisterSlot( regNum, pRD );
-#if defined(TARGET_UNIX) && !defined(FEATURE_NATIVEAOT) && !defined(SOS_TARGET_AMD64)
+#if defined(TARGET_UNIX) && !defined(FEATURE_NATIVEAOT) && !defined(SOS_TARGET_RISCV64)
```

Comment:
Was replacing ARM64 with RISCV64 intentional or do we need both?

Reply:
Yes, they were incorrect and caused build problems in the diagnostic repo. This section of code is under ifdef TARGET_RISCV64.